spec/system/admin/admin_company_value_crud_system_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin Company Value management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:admin_user) { create(:admin_user) }
+  let(:new_company_value) { build(:company_value) }
+  let(:edited_company_value) { build(:company_value) }
+
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/MultipleExpectations
+
+  it 'enables me to create, read, update and destroy Company Value' do
+    login_as(admin_user)
+    visit admin_company_values_path
+
+    find_link('Create New Company Value!').click
+    fill_in 'company_value[title]', with: new_company_value.title
+    click_button 'Create Company value'
+    expect(page).to have_text('Company Value was successfully created.')
+    expect(page).to have_text(new_company_value.title)
+
+    find_link('Edit...', match: :first).click
+    fill_in 'company_value[title]', with: edited_company_value.title
```

Comment:
it's good practice to be consequent with your usage of `(` - L#28 vs L#27 etc.

Reply:
I have corrected some of the specification files. I'll try to review them more closely in relation to your hint and improve them in the near future.